vmdk_plugin/refcnt.go

```diff
@@ -170,10 +195,14 @@ func (r refCountsMap) getCount(vol string) uint {
 
 // incr refCount for the volume vol. Creates new entry if needed.
 func (r refCountsMap) incr(vol string) uint {
-	rc := r[vol]
+	// Locks the refCountsMap
```

Comment:
Same question. There seems to be a gap between the refcount is changed in incr() and checked in Mount().  During this time another thread can change the value